src/BuildServer/Services/ArtifactBuilders/ArtifactBuilder.cs

```diff
@@ -48,7 +48,9 @@ public bool Build(Build build)
             };
             try
             {
+#pragma warning disable CA1416 // Validate platform compatibility
```

Comment:
Could you please confirm: do we know on which platform this code would be invalid?